tests/unit/utils.ts

```diff
@@ -90,6 +92,7 @@ export function generateCompletionData(
   tstText: string,
   position: CompletionPosition,
   triggerCharacter: string | undefined,
+  cppTest: { cppNode: boolean; lineSoFar: string },
```

Comment:
Let's make this be an interface, so we can have nullable args and not worry about always passing all of this stuff through.

Reply:
Made an interface of the optional params in 9038de54b41c65eec73f8f620cbae868eb5a3936 .